requirements.txt

```diff
@@ -1,3 +1,3 @@
 PyYAML
-PyGithub
+PyGithub @ git+https://github.com/txqueuelen/PyGithub@add-missing-attributes
```

Comment:
I am using a fork until PyGithub/PyGithub#2512 is merged.

Reply:
If possible I would wait a couple days until this is reviewed (if we're lucky, merged) upstream just as an additional chance to catch mistakes on the fork.

Reply:
I moved the reference from branch to commit [here](https://github.com/txqueuelen/reposettings/pull/11/commits/a28c009b5281fdde8e6c21ea30a80a2cbdbbbade)

So we can merge it right now and see how the PR goes because PyGihub takes months to merge PRs.